libraries/postgres/src/Database.js

```diff
@@ -186,7 +186,9 @@ class Database {
         showProgress('...checking permissions');
         await Database._checkPermissions({ db, schema, usernamePrefix });
         showProgress('...checking table columns');
-        await Database._checkTableColumns({ db, schema, usernamePrefix });
+        await Database._checkTableColumns({db, schema, usernamePrefix});
+        showProgress('...checking version');
+        await Databse._checkVersion({db, schema});
```

Comment:
The comment above this section indicates this is only run at the end of the update.  There are some calls to `this._checkXxx` at the top of this function which run on *every* update, before it starts, and that might be a better place to put this.

Reply:
Also, [it looks like the test failed](https://community-tc.services.mozilla.com/tasks/PG2wotIHRCW32vasmU4vIA/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FPG2wotIHRCW32vasmU4vIA%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L569)

Reply:
The error message shows, function does not exist, but it does

Reply:
I forgot to make it static XD